BedrockServer.cpp

```diff
@@ -558,14 +558,17 @@ void BedrockServer::worker(SData& args,
                     // Peek wasn't enough to handle this command. Now we need to decide if we should try and process
                     // it, or if we should send it off to the sync node.
                     bool canWriteParallel = false;
-                    {
-                        SAUTOLOCK(_parallelCommandMutex);
+                    // Multi-write needs to be enabled.
+                    canWriteParallel = server._multiWriteEnabled.load();
```

Comment:
Can we just initialize `canWriteParallel` with `server._multiWriteEnabled.load()` and thus combine this and the line above?